glossary.html

```diff
@@ -69,24 +42,9 @@ <h1>Acronym Glossary</h1>
     
     
     <footer>
-      <style>
-      p.ex1 {
-    margin-bottom: 2em;
-        }
-      
-        div hr{
```

Comment:
Use `footer hr` instead of `div hr` to select only `hr` elements in the footer.  The selector you have selects any hr elements in any div